components/EditFile.tsx

```diff
@@ -23,13 +23,15 @@ export const EditIcon = () => (
 const EditFile: React.FC<Props> = ({ slug }) => {
     const url = `https://github.com/100mslive/100ms-docs/blob/main/docs${slug}.mdx`;
     return (
-        <div>
+        <div style={{ width: 'fit-content' }}>
```

Comment:
Minor Nit: can we add it in `<style jsx />` instead of inlining it.

Also didn't @hardik1003 added tailwind css 😬

Reply:
No tailwind here 🥲